examples/1_hw_basic_control.py

```diff
@@ -1,9 +1,7 @@
-import time
```

Comment:
Given this is an example, I would either add a readme with a one-liner about what this examples exhibits or a docstring at the top of script.